public/fullSchema.json

```diff
@@ -169,6 +169,76 @@
           }
         }
       }
+    },
+    "/accounts/{account_id}/sites/{site_identifier}/notifiable_users": {
```

Comment:
I think we can reuse `sites/{site_identifier}/users` for that purpose since it is not currently used to anything and I can see it would serve the same purpose

Reply:
agree with Robert